--- a/dwm.c	2020-09-21 22:19:26.375461850 +0200
+++ b/dwm.c	2020-09-21 22:18:38.642127316 +0200
@@ -882,6 +882,7 @@ focus(Client *c)
unfocus(selmon->sel, 0);
if (c)
{
+        c->bw = borderpx;
if (c->mon != selmon)
selmon = c->mon;
if (c->isurgent)
@@ -891,6 +892,7 @@ focus(Client *c)
grabbuttons(c, 1);
XSetWindowBorder(dpy, c->win, scheme[SchemeSel][ColBorder].pixel);
setfocus(c);
+        arrange(c->mon);
}
else
{
@@ -1983,6 +1985,9 @@ unfocus(Client *c, int setfocus)
XSetInputFocus(dpy, root, RevertToPointerRoot, CurrentTime);
XDeleteProperty(dpy, root, netatom[NetActiveWindow]);
}
+
+    c->bw = 0;
+    arrange(c->mon);
}
void
